Modules/Quotas/src/Quotas.Infrastructure/Persistence/Repository/RelationshipsRepository.cs

```diff
@@ -19,7 +19,8 @@ public async Task<uint> Count(string createdBy, DateTime createdAtFrom, DateTime
         var relationshipsCount = await _relationshipsReadonly
             .CreatedInInterval(createdAtFrom, createdAtTo)
             .Where(r => r.Status == RelationshipStatus.Pending && r.From == createdBy ||
-                r.Status == RelationshipStatus.Active && (r.From == createdBy || r.To == createdBy))
+                r.Status == RelationshipStatus.Active && (r.From == createdBy || r.To == createdBy) ||
+                r.Status == RelationshipStatus.Terminated && r.AuditLog.Last().Reason == RelationshipAuditLogEntryReason.Reactivation)
```

Comment:
Relationships with requested termination should only count for the quotas of the identity that requested reactivation.

Reply:
Changes made.

Reply:
This is still not fixed. With your implementations, a relationship for which reactivation was requested, counts towards the quota for both participants. But it should only count for the identity that requested the reactivation.

Reply:
This part of the code is outdated, the changes where indeed added. When I go to the files this is shown. I hope this is what you wanted to me fix.
![image](https://github.com/nmshd/backbone/assets/117634437/f3b17b23-56cf-44cf-92a3-812535d9eccc)

Reply:
Changes made. Now it should correctly count.